academy/4-my-own-chain/cosmjs-objects.md

```diff
@@ -11,53 +11,53 @@ tag: deep-dive
 
 Make sure you have everything you need before proceeding:
 
-* You understand the concepts of [Protobuf](../2-main-concepts/protobuf.md), and [CosmJs](TODO).
-* Have Go and npm installed.
-* The checkers blockchain codebase up to the wager denomination. You can get there by following the [previous steps](./wager-denom.md) or checking out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/wager-denomination).
+* You understand the concepts of [Protobuf](../2-main-concepts/protobuf.md) and [CosmJs](TODO).
+* Go and npm are installed.
+* You have the checkers blockchain codebase up to the wager denomination. If not, follow the [previous steps](./wager-denom.md) or check out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/wager-denomination).
 
 </HighlightBox>
 
-With your Checkers application ready for use, it is a good time to prepare its client elements that will eventually allow you to create a GUI and/or server-side scripts. Here, you are going to apply [what you learned](TODO) about creating your own custom CosmJs interfaces.
+With your Checkers application ready for use, it is a good time to prepare client elements that will eventually allow you to create a GUI and/or server-side scripts. Here, you will apply [what you learned](TODO) about creating your own custom CosmJs interfaces.
 
 ## Compile Protobuf
 
 Create a `client` folder that will contain all these new elements.
 
-In fact, if you want to keep the Go parts of your Checkers project separate from the Typescript parts, you might want to use another repository for the _client_. In order to keep a link between the two repositories, you can add the _client_ parts as a submodule to your Go parts:
+If you want to keep the Go parts of your Checkers project separate from the Typescript parts, you can use another repository for the _client_. To keep a link between the two repositories, add the _client_ parts as a submodule to your Go parts:
 
 ```sh
 $ git submodule add git@github.com:cosmos/academy-checkers-ui.git client
 ```
 
 Replace the path with your own repository. In effect, this creates a new `client` folder. This `client` folder makes it possible for you to easily update another repository with the content of your Go code.
 
-Create another folder named `script` in your project root. It is from here that you will launch the Protobuf compilation. In it, install modules for the Protobuf-to-Typescript compiler:
+Create a folder named `script` in your project root. This is where you will launch the Protobuf compilation. In the `script` folder install modules for the Protobuf-to-Typescript compiler:
```

Comment:
```suggestion
Create a folder named `script` in your project root. This is where you will launch the Protobuf compilation. In the `scripts` folder install modules for the Protobuf-to-Typescript compiler:
```

Reply:
To confirm: there are two different folders called "script" and "script**s**" respectively?

Reply:
Oops, I updated my suggestion: `scripts` everywhere.
When you are satisfied, you can commit the suggestions in a batch from the "Files changed" tab.